run.sh

```diff
@@ -0,0 +1,7 @@
+set -e
+set -x
+
+jekyll build
+aws s3 sync ./_site s3://quay-docs --delete --acl public-read
+aws configure set preview.cloudfront true
+aws cloudfront create-invalidation --distribution-id E26ZSD57Z81FLL --invalidation-batch "{\"Paths\":{\"Quantity\":1,\"Items\":[\"/*\"]},\"CallerReference\":\"jenkins-$(date)\"}"
```

Comment:
I've changed it to a var, but it's still out here in this PR. I can't find documentation on whether or not it's a secret.